CONTRIBUTING.md

```diff
@@ -59,8 +59,7 @@ The tests are currently console application projects. Simply run the following c
 
 ```Text
 cd src/NodaTime.Test
-dotnet run -f net451
-dotnet run -f netcoreapp1.0
+dotnet run -f netcoreapp2.0
```

Comment:
Just `dotnet run` should be fine now.

Reply:
Okido, will fix (and double check), gimme a sec.

Reply:
Worked like a charm, much simpler indeed. Fixed this.

(PS. I also found a second instance of this text for the Web files, changed the same thing there as well. Wasn't ready for the blitz-review of my PR! :smile:)